cpp/tests/io/parquet_test.cpp

```diff
@@ -266,7 +266,7 @@ TYPED_TEST(ParquetWriterChronoTypeTest, ChronosWithNulls)
   auto filepath = temp_env->get_temp_filepath("ChronosWithNulls.parquet");
   cudf_io::parquet_writer_options out_opts =
     cudf_io::parquet_writer_options::builder(cudf_io::sink_info{filepath}, expected->view());
-  cudf_io::write_parquet(out_opts);
```

Comment:
Should we also keep the old API for non-chunked writes? It is, after all, consistent with other formats' APIs. Also avoid fully breaking the API.

Reply:
reverted.